ports/rp2/boards/PI-PLATES_MICROPI/board.json

```diff
@@ -0,0 +1,25 @@
+{
+    "deploy": [
+        "../deploy.md"
+    ],
+    "docs": "",
+    "features": [
+        "Pi-Plate Compatible",
+        "Data Acquisition",
+        "Micro USB",
+        "Real Time Clock",
+        "OLED Display",
+        "Temperature Sensor",
+        "LED Annunciators",
+        "microSD Storage"
+    ],
+    "id": "MICROpi",
+    "images": [
+        "MICROpi.jpg"
+    ],
+    "mcu": "rp2040",
+    "product": "MICROpi",
+    "thumbnail": "",
+    "url": "https://www.Pi-Plates.com/MICROpi/",
```

Comment:
The url appears to be invalid; maybe the MICROpi hasn't been released yet?

Reply:
Correct. Current release date is 9/21